platform/macos/export/export_plugin.cpp

```diff
@@ -2051,13 +2051,17 @@ bool EditorExportPlatformMacOS::has_valid_export_configuration(const Ref<EditorE
 	String architecture = p_preset->get("binary_format/architecture");
 	if (architecture == "universal" || architecture == "x86_64") {
 		if (!ResourceImporterTextureSettings::should_import_s3tc_bptc()) {
+			err += TTR("Cannot export for universal or x86_64 if S3TC BPTC texture format is disabled. Enable it in the Project Settings (Rendering > Textures > VRAM Compression > Import S3TC BPTC).") + "\n";
```

Comment:
> Enable it in the Project Settings [...]

Instead of requiring the user to dig into the settings, we should just provide a button to fix it. I implemented this in PR #78457, but the button seems to have disappeared somewhere in-between my PR and your PR? It's still present for Android, but missing for macOS for some weird reason.